.travis.yml

```diff
@@ -8,15 +8,12 @@ install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
   - go get -u golang.org/x/net/context
-  - go get -u github.com/HewlettPackard/gas
   - go get -u gopkg.in/godo.v2/cmd/godo
   - export GO15VENDOREXPERIMENT=1
   - glide install
 
 script:
-  - gas -skip=*/arm/*/models.go -skip=*/management/examples/*.go -skip=*vendor* -skip=*/Gododir/* ./...
-  - gas -exclude=G101 ./arm/... ./management/examples/...
-  - gas -exclude=G204 ./Gododir/...
+  - bash rungas.sh
```

Comment:
Well, after quickly perusing the Travis Docs the closest that I found was [enabling or disabling all CI on branches](https://docs.travis-ci.com/user/customizing-the-build#Safelisting-or-blocklisting-branches). If you find a better way, just give me a holler. :)